src/mpl/src/gpu/mpl_gpu_ze.c

```diff
@@ -387,6 +388,20 @@ int MPL_gpu_init_device_mappings(int max_devid, int max_subdevid)
         global_to_local_map[i] = -1;
     }
 
+    /* Initialize global_to_root_map */
```

Comment:
Expand the comment to add its purpose (to support for the case when querying a device which is not visible to
the current rank due to ZE_AFFINITY_MASK).

Reply:
Added